uiowa/uiowa
# [Icons] Card block

# How to test

```
ddev blt frontend && ddev drush smw-svg  && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /node/21/layout
```

1. Review https://github.com/uiowa/uids/pull/948
2. Code review
3. Test adding icons to different card variants
4. Page through the entire library of icons and make sure none of the thumbnails are broken or appear off-centered
5. Test search experience in media library grid/table list alpha, newest, etc.



## How to add new icons in the future
1. Attach new hash from https://github.com/uiowa/brand-icons to uids_base `package.json`
2. Run `ddev blt frontend && ddev drush smw-svg` and commit the new files
3. Attach update hook to `sitenow_media_wysiwyg.install` to run `sitenow_media_wysiwyg_media_import_icons()` function.
